pkgs/development/tools/wasmserve/default.nix

```
@@ -0,0 +1,21 @@
+{ buildGoModule
+, lib
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "wasmserve";
+  version = "1.0.0";
+  src = fetchFromGitHub {
+    owner = "hajimehoshi";
+    repo = "wasmserve";
+    rev = "v${version}";
+    sha256 = "sha256-KlCbUre6yIorE1ZM++Rto8vgwVGsC1wZj1xCd3AwQy0=";
+  };
+  vendorHash = null;
+  meta = with lib; {
+    description = "An HTTP server for testing Wasm";
+    homepage = "https://github.com/hajimehoshi/wasmserve";
+    license = licenses.asl20;
+    maintainers = [ maintainers.kirillrdy ];
+  };
+}
```

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
}:

buildGoModule rec {
  pname = "wasmserve";
  version = "1.0.0";

  src = fetchFromGitHub {
    owner = "hajimehoshi";
    repo = "wasmserve";
    rev = "v${version}";
    hash = "sha256-KlCbUre6yIorE1ZM++Rto8vgwVGsC1wZj1xCd3AwQy0=";
  };

  vendorHash = null;

  meta = with lib; {
    description = "An HTTP server for testing Wasm";
    homepage = "https://github.com/hajimehoshi/wasmserve";
    license = licenses.asl20;
    maintainers = with maintainers; [ kirillrdy ];
  };
}
Commenting
```

just some formatting nitpicks. 
btw do you know [nix-init](https://github.com/nix-community/nix-init)? would have probably work great for this.

---

formatting changes make sense to me,

I have seen nix-init although never used it, will give it a try ! thanks !